hack/common.sh

```diff
@@ -815,15 +815,17 @@ function os::build::image() {
   local dockerfile="${3-}"
   local options="${OS_BUILD_IMAGE_ARGS-}"
   local mode="${OS_BUILD_IMAGE_TYPE:-imagebuilder}"
+
   if [[ "${mode}" == "imagebuilder" ]]; then
-    if [[ -n "$( which imagebuilder )" ]]; then
+    if os::util::find::system_binary 'imagebuilder'; then
       if [[ -n "${dockerfile}" ]]; then
         eval "imagebuilder -f '${dockerfile}' -t '${tag}' ${options} '${directory}'"
```

Comment:
@stevekuznetsov, yeah, the user just needs to use normal shell escaping as necessary.  For example, something like `OS_BUILD_IMAGE_ARGS='-foo=bar\ baz -bar -baz="quux xyzzy"'` should work as expected.